lib/client.js

```diff
@@ -351,6 +351,15 @@ Client.prototype.registerIBAN = function (ip, _opts) {
   return this._request('RegisterIBAN', ip, opts).get('IBAN');
 };
 
+Client.prototype.createVirtualIBAN = function(ip, _opts) {
```

Comment:
```suggestion
Client.prototype.createVirtualIban = function(ip, _opts) {
```

? to follow internal guidelines

Reply:
It was to be coherent with existing methods in the package `registerIBAN`, `createVCC`, `moneyInSDDInit`, `unregisterSDDMandate`, `registerSDDMandate` 🙈

Reply:
Ok Ok …